packages/web/src/components/notification/Notifications/UserNameLink.module.css

```diff
@@ -1,5 +1,5 @@
-.root {
-  margin-right: 4px;
+.rootLeftMargin {
```

Comment:
to support RTL, should this be `margin-inline-start`?

Reply:
ohh that is genius great catch